cmd/tendermint/commands/version.go

```diff
@@ -11,8 +12,20 @@ import (
 // VersionCmd ...
 var VersionCmd = &cobra.Command{
 	Use:   "version",
-	Short: "Show version info",
+	Short: "Show protocols' and libraries' version numbers",
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(version.TMCoreSemVer)
+		values, _ := json.MarshalIndent(struct {
+			Tendermint    string
+			ABCI          string
+			BlockProtocol uint64
+			P2PProtocol   uint64
+		}{
```

Comment:
Is the idea with the JSON format that this is going to be machine-interpreted? Who or what would be interpreting it?

And why the difference in JSON field naming convention as compared to what's used, for example, in the genesis file (`Tendermint` as opposed to `tendermint`, and `BlockProtocol` vs `block_protocol`)?

Reply:
did json format cause otherwise it looks ugly when printed. I can Rename if needed, I see this way as nicer

Reply:
I have a few suggestions here:
1. Let's please rather keep the current `version` command default behaviour as-is (just printing the Tendermint version). I don't know if we _always_ need to see all versions of all protocols in that version.
2. Could we add a `-v`/`--verbose` flag to the `version` command to print this JSON struct?
3. Let's please stick to the current JSON conventions used throughout the codebase, which favour `snake_case`.

Reply:
Made the respective updates. 

I don't think this requires a changelog as it doesn't really change the behaviour in a notable way but let me know if you disagree